tests/toy.rs

```diff
@@ -57,3 +58,56 @@ fn test_toy_index() -> anyhow::Result<()> {
     );
     Ok(())
 }
+
+#[test]
+fn test_to_and_from_ciff() -> anyhow::Result<()> {
+    let input_path = PathBuf::from("tests/test_data/toy-complete-20200309.ciff");
+    let temp = TempDir::new().unwrap();
+    let output_path = temp.path().join("coll");
+    match ciff_to_pisa(&input_path, &output_path) {
+        Err(error) => panic!("{}", error),
+        Ok(_) => {}
+    }
+    let ciff_output_path = temp.path().join("ciff");
+    pisa_to_ciff(
+        &output_path,
+        &temp.path().join("coll.terms"),
+        &temp.path().join("coll.documents"),
+        &ciff_output_path,
+        "Export of toy 3-document collection from Anserini's io.anserini.integration.TrecEndToEndTest test case",
+    )?;
+
+    // NOTE: the constructed ciff file will not be exactly the same as the initial one.
+    // The reason is that PISA index will be treated as a whole index while the statistics
+    // in the initial ciff are as if it is a chunk of a larger index. Therefore, we convert
+    // back to PISA to verify.
+
+    let pisa_copy = temp.path().join("copy");
+    ciff_to_pisa(&ciff_output_path, &pisa_copy)?;
```

Comment:
Yes, it makes sense, but it would be great to have a better toy index. I dont expect to address it in this PR.

Reply:
Now that we have it both ways, we can generate a binary collection from pisa and use that as another test. Feel free to open a ticket or PR.